### DISCUSSION TITLE
Support for Admin Log, Roles and Permission

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo_admin#200] [Ideas]
One nice feature of Django is the Admin Log,
Another great feature is Roles and Permission. 
If Piccolo can support these then it will be very helpful.

A humble request to also support for object level permission.
Django doesn't provide it by default.  I had to use the additional package `django-guardian`.

**Comment by sinisaos** [comment]
@sumitsharansatsangi Audit logs are under development as a separate Piccolo application (PR is [here](https://github.com/piccolo-orm/piccolo_api/pull/160)), but first we need to merge this [PR](https://github.com/piccolo-orm/piccolo_api/pull/145) to be able to get the correct logs for batch actions. Once we've sorted out the Piccolo API, we need to make changes to the Piccolo Admin (we have almost all the code ready). I hope it will be soon.

**Reply by sumitsharansatsangi** [reply]
Nice to know that it is already in development.
